client/src/App.js

```diff
@@ -114,8 +114,36 @@ export default class NormalDisplay extends React.Component {
         }
     }
 
+    async getJournalUrl() {
+        if (this.isAborted) return;
+        let resp;
+        try {
+            resp = await fetch("/api/get-journal-url", {
+                method: 'GET',
+                headers: {
+                    Authorization: this.state.devToolToken
+                }
+            });
+            if (!resp.ok) {
+                const errorMessage = await this.formatErrorMessage(resp, 'Error setting up event provider');
+                throw new Error(errorMessage);
+            }
+            resp = await resp.json();
+            console.log(`Asset Compute journal url: ${resp.journalUrl}`);
```

Comment:
use `Log(message)` instead of `console.log` to log things. It is a logger that only displays in the console when debug mode is on.